.github/workflows/docs.yml

```diff
@@ -11,13 +11,18 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions:
+      actions: write
     steps:
       - uses: actions/create-github-app-token@v1
         id: docs-github-app-token
         with:
           app-id: ${{ vars.DOCS_APP_ID }}
           private-key: ${{ secrets.DOCS_APP_SECRET_KEY }}
-      - uses: actions/checkout@v4
+          owner: umts
+          repositories: dev-training-web
+          permission-contents: write
```

Comment:
Pretty much a belt-and-suspenders setup. The OAuth app goes with this secret already only has 
read/write repository contents" permissions and is only installed in this one repo.